test/TodoTest.java

```diff
@@ -0,0 +1,10 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TodoTest {
+    @Test
+    void testToString() {
+    Todo newTodo = new Todo("Read Asura Omega manga");
+    assertEquals("[T][✘] Read Asura Omega manga",newTodo.toString());
```

Comment:
These lines should be tabbed in, shouldn't they?

Reply:
I have gone through all of your comments and I found then really helpful. Thank you for making the effort to go through all the fine details, I appreciate it 👍